app/views/about/index.html.erb

```diff
@@ -32,6 +33,7 @@
                   <td><%= link_to_if_authorized proxy.name, hash_for_edit_smart_proxy_path(proxy) %></td>
                   <td><%=h proxy.features.to_sentence %></td>
                   <td><div class="proxy-status" data-url=<%= ping_smart_proxy_path(proxy) %>><%= _("Connecting..") %></div></td>
+                  <td><div class="proxy-version" data-url=<%= version_api_smart_proxy_path(proxy) %>><%= _("Connecting..") %></div></td>
```

Comment:
Avoid using the API within the UI for AJAX, we don't do this usually - the separate route you had before was fine.